### PR TITLE
Changed name of service

### PR DESCRIPTION
**Pull request by RobCatton**
# Description

-   To change all instances of Fares Data Build Tool to Create Fares Data or Create Fares Data Service

# Testing instructions

-   Search repo for Fares Data Build Tool instances.

# Type of change

-   [ ] feat - A new feature
-   [ ] fix - A bug fix
-   [ ] docs - Documentation only changes
-   [ ] codestyle - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] refactor - A code change that neither fixes a bug nor adds a feature
-   [ ] perf - A code change that improves performance
-   [ ] test - Adding missing tests or correcting existing tests
-   [ ] build - Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
-   [ ] ci - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] chore - Other changes that don't modify src or test files
-   [ ] revert - Reverts a previous commit
-   [x] content - Changes to content or copy

# Checklist:

-   [x] Able to run pr locally
-   [x] Followed acceptance criteria
-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have made corresponding changes to the documentation if applicable
-   [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: src/pages/accessibility.tsx [hunk 1]
```diff
@@ -1,19 +1,19 @@
 import React, { ReactElement } from 'react';
 import { TwoThirdsLayout } from '../layout/Layout';
 
-const title = 'Accessibility - Fares Data Build Tool';
-const description = 'Accessibility page for the Fares Data Build Tool';
+const title = 'Accessibility - Create Fares Data';
```
[the remark this answers is not in the document]
Comment (RobCatton) [reply src/pages/accessibility.tsx:4]: Fixed.

File: src/pages/accessibility.tsx [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
         <TwoThirdsLayout title={title} description={description}>
-            <h1 className="govuk-heading-l">Accessibility statement for the Fares Data Build Tool</h1>
+            <h1 className="govuk-heading-l">Accessibility statement for the Create Fares Data Service</h1>
             <p className="govuk-body">
                 This service is part of the wider GOV.UK website. There’s a separate accessibility statement for the
                 main GOV.UK website.
             </p>
             <p className="govuk-body">
-                This page only contains information about the Fares Data Build Tool service, available at&nbsp;
+                This page only contains information about the Create Fares Data Service service, available at&nbsp;
```
[the remark this answers is not in the document]
Comment (RobCatton) [reply src/pages/accessibility.tsx:16]: Fixed

File: src/pages/contact.tsx [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
+const title = 'Contact - Create Fares Data Service';
+const description = 'Contact page for the Create Fares Data Service';
 
 const Contact = (): ReactElement => {
     return (
         <BaseLayout title={title} description={description}>
             <div className="govuk-grid-row">
                 <div className="govuk-grid-column-two-thirds">
-                    <h1 className="govuk-heading-l">Contact the Fares Data Build Tool team</h1>
+                    <h1 className="govuk-heading-l">Contact the Create Fares Data Service team</h1>
```
[the remark this answers is not in the document]
Comment (RobCatton) [reply src/pages/contact.tsx:12]: Can do, could see it either way. Changed.

File: src/pages/home.tsx [hunk 4]
```diff
@@ -3,16 +3,16 @@ import { NextPageContextWithSession, CustomAppProps } from '../interfaces';
 import { BaseLayout } from '../layout/Layout';
 import { checkIfMultipleOperators } from '../utils';
 
-const title = 'Fares Data Build Tool';
-const description = 'Fares Data Build Tool is a service that allows you to generate data in NeTEx format';
+const title = 'Create Fares Data Service';
+const description = 'Create Fares Data Service is a service that allows you to generate data in NeTEx format';
```
[the remark this answers is not in the document]
Comment (RobCatton) [reply src/pages/home.tsx:7]: yeah good catch, changed.

File: src/pages/home.tsx [hunk 5]
```diff
@@ -3,16 +3,16 @@ import { NextPageContextWithSession, CustomAppProps } from '../interfaces';
 import { BaseLayout } from '../layout/Layout';
 import { checkIfMultipleOperators } from '../utils';
 
-const title = 'Fares Data Build Tool';
-const description = 'Fares Data Build Tool is a service that allows you to generate data in NeTEx format';
+const title = 'Create Fares Data Service';
```
[the remark this answers is not in the document]
Comment (RobCatton) [reply src/pages/home.tsx:6]: Fixed.

File: src/pages/index.tsx [hunk 6]
```diff
@@ -3,16 +3,16 @@ import TwoThirdsLayout from '../layout/Layout';
 import { CustomAppProps, NextPageContextWithSession } from '../interfaces';
 import { checkIfMultipleOperators } from '../utils';
 
-const title = 'Fares Data Build Tool';
-const description = 'Fares Data Build Tool is a service that allows you to generate data in NeTEx format';
+const title = 'Create Fares Data Service';
```
[the remark this answers is not in the document]
Comment (RobCatton) [reply src/pages/index.tsx:6]: Fixed

File: src/pages/index.tsx [hunk 7]
```diff
@@ -3,16 +3,16 @@ import TwoThirdsLayout from '../layout/Layout';
...
+const description = 'Create Fares Data Service is a service that allows you to generate data in NeTEx format';
```
[the remark this answers is not in the document]
Comment (RobCatton) [reply src/pages/index.tsx:7]: Fixed

File: src/pages/login.tsx [hunk 8]
```diff
@@ -32,7 +32,7 @@ const Login = ({ errors = [], csrfToken, email }: LoginProps & CustomAppProps):
                                     </h1>
                                 </legend>
                                 <p className="govuk-hint hint-text">
-                                    Enter Fares Data Build Tool account details to sign in
+                                    Enter Create Fares Data account details to sign in
```
[the remark this answers is not in the document]
Comment (RobCatton) [reply src/pages/login.tsx:35]: I think it specifically calls it out so people dont try sign in with BODS credentials.

File: src/pages/requestAccess.tsx [hunk 9]
```diff
@@ -11,11 +11,11 @@ const RequestAccess = (): ReactElement => (
                 <h1 className="govuk-heading-l" id="request-access-page-heading">
                     Request Access
                 </h1>
-                <p className="govuk-body">The Fares Data Build Tool is not open to public registration.</p>
+                <p className="govuk-body">The Create Fares Data Service is not open to public registration.</p>
                 <p className="govuk-body">
                     Please contact{' '}
                     <a className="govuk-link" id="access_link" href="/contact">
-                        Fares Data Build Support
+                        Create Fares Data support
```
[the remark this answers is not in the document]
Comment (RobCatton) [reply src/pages/requestAccess.tsx:18]: changed.

**Review by dannydaviesIW — APPROVED** [review]

**Pull request merged**